docs/ref/models/querysets.txt

```diff
@@ -1635,14 +1635,23 @@ queryset is evaluated. You can also ignore locked rows by using
 ``select_for_update()`` with both options enabled will result in a
 :exc:`ValueError`.
 
+By default, ``select_for_update()`` locks all rows that are selected by the
+query. For example, rows of related objects specified in
+:meth:`~django.db.models.query.QuerySet.select_related()` are also locked,
+in addition to rows of the base model. If this is not desirable, you can
+specify the specific related objects you want to lock in
+``select_for_update(of={...})``, similarly to
```

Comment:
I think that we should be consistent with `of` type and use `tuple` everywhere (including docs, tests and examples).